### DISCUSSION TITLE
Nuxt Module Intellisense Support

### DISCUSSION THREAD

**Discussion by yuyinws** [nuxtrdev/nuxtr-vscode#144] [Ideas]
I develop a [vscode extensions](https://github.com/yuyinws/nuxt-module-intellisense) to help user autocomplete Nuxt Module.

![a](https://camo.githubusercontent.com/b69d840637b5f8016b817064c460b64553c9f777c2b18e33ec50e9ddd00b136d/68747470733a2f2f63646e2e6a7364656c6976722e6e65742f67682f797579696e77732f737461746963406d61737465722f323032342f30342f75706769745f32303234303432395f313731343337393635342e706e67)

I think it's a good idea to integrate it into Nuxtr.  And I am willing to send a PR for it. WDYT?

**Comment by adhamfarrag** [comment]
Nice work here @yuyinws and thanks for considering contribution and adding the feature to Nuxtr. ❤️
I'd love to add it! But can you give me couple weeks? need to change couple things that will directly affect the implementation.

**Reply by yuyinws** [reply]
Sure, you can reply to me here when everything is ready. 😃
